src/components/paragraph/rows/one-column.tsx

```diff
@@ -14,7 +14,7 @@ const OneColumn = async ({items, fullWidth = true}: LayoutProps) => {
     draftProps["data-columns"] = "1"
   }
   return (
-    <div className="space-y-16 @container" {...draftProps}>
+    <div className="flex flex-col space-y-90" {...draftProps}>
```

Comment:
no need for flex classes if you use the space class right? either space or flex + gap

Reply:
🤔 I couldn't seem to get the `space-y-*` working without adding flex but I'll update the styles to be consistent to what SUL had before (with `gap-90`)

Reply:
`space-y-90` does work for me without the flex classes. It up to you, either way works for me.

Reply:
I think part of the issue is the children inside might also have their own margin top/bottom which will conflict with the space classes.